app/views/wpcc/your_details/new.html.erb

```diff
@@ -111,43 +125,46 @@
+                  <%= t('wpcc.details.calculate.qualifying_earnings_title') %>:
+                </span>
+                <%= t('wpcc.details.calculate.qualifying_earnings') %>
+              </p>
+              <%= render 'wpcc/tooltips/close' %>
+            </div>
 
-              <div class="details__callouts">
-                <div class="form__row details__callout <%= @your_details_form.activate_disabled_callout %>" data-dough-callout-radio-disabled>
-                  <div class="callout">
-                    <p><%= t('wpcc.details.callout__radiodisabled') %></p>
-                  </div>
+            <div class="details__callouts">
+              <div class="form__row details__callout <%= @your_details_form.activate_disabled_callout %>" data-dough-callout-radio-disabled>
+                <div class="callout">
+                  <p><%= t('wpcc.details.callout__radiodisabled') %></p>
                 </div>
               </div>
-              <div class="form__group-item details__calculate-item <%= @your_details_form.error_class('contribution_preference') %>">
-                <%= f.radio_button(:contribution_preference, 'minimum', class: 'details__calculate-item-select', 'data-dough-employer-part-radio': true) %>
-                <%= f.label(:contribution_preference_minimum, t('wpcc.details.options.contribution_preference.minimum'), class: 'details__calculate-item-label') %>
-              </div>
-              <div class="form__group-item details__calculate-item">
-                <%= f.radio_button(:contribution_preference, 'full', class: 'details__calculate-item-select', 'data-dough-employer-full-radio': true) %>
-                <%= f.label(:contribution_preference_full, t('wpcc.details.options.contribution_preference.full'), class: 'details__calculate-item-label') %>
-              </div>
-            </fieldset>
-          </div>
+            </div>
+            <div class="details__row details__row--group">
+              <%= f.form_row :contribution_preference do %>
+                <%= f.errors_for :contribution_preference %>
+                <div class="form__group-item details__calculate-item">
+                  <%= f.radio_button(:contribution_preference, 'minimum', class: 'details__calculate-item-select', 'data-dough-employer-part-radio': true) %>
+                  <%= f.label(:contribution_preference_minimum, t('wpcc.details.options.contribution_preference.minimum'), class: 'details__calculate-item-label') %>
+                </div>
+                <div class="form__group-item details__calculate-item">
+                  <%= f.radio_button(:contribution_preference, 'full', class: 'details__calculate-item-select', 'data-dough-employer-full-radio': true) %>
+                  <%= f.label(:contribution_preference_full, t('wpcc.details.options.contribution_preference.full'), class: 'details__calculate-item-label') %>
+                </div>
+              <% end %>
+            </div>
+          </fieldset>
```

Comment:
@tomas-stefano there's no PR for dealing with other fields. I will go ahead and remove them still.